cmd/acra-key-tool/acra-key-tool.go

```diff
@@ -36,8 +40,8 @@ import (
 )
 
 var (
-	serviceName       = "acra-read-key"
-	defaultConfigPath = utils.GetConfigPathByName("acra-read-key")
+	serviceName       = "acra-key-tool"
```

Comment:
I'm not particularly attached, but I do have [some aversion](https://blog.codinghorror.com/i-shall-call-it-somethingmanager/) to naming things with a “manager” suffix (or anything with “-er” verb nouns, really) instead of the action directly.

With acra-read-key it's straightforward that it reads keys, and that would be perfectly fine if it does only that. Like acra-rotate only rotates data. But with this... we either introduce a flock of tools, each with a specific purpose (UNIX-way, yay! @gene-eu may like it) Or we need to invent a better name. I doubt there is any *objectively better* name for a tool that will do *everything* related to keys. [acra-key-ramirez](https://knowyourmeme.com/memes/ramirez-do-everything), anyone?